Corcav.Behaviors/Library/EventToCommand.cs

```diff
@@ -148,7 +148,7 @@ private void AddEventHandler(EventInfo eventInfo, object item, Action<EventArgs>
         /// <param name="e">The EventArgs value accompanying the Event</param>
         private void OnFired(EventArgs e)
 		{
-            object returnParam = this.CommandParameter ?? e;
+            object returnParam = e ?? this.CommandParameter;
```

Comment:
@snickler I would appreciate a property to determine if EventArgs are needed. But I also liked the way it was before. To have the value of CommandParameter as first option and the  EventArgs only if CommandParameter is null. 
I think an additional property is more clear for the user and gives more control.